src/pages/getting-started/app-guides/chat-webhooks/index.mdx

```diff
@@ -4,14 +4,12 @@ category: "getting-started"
 subcategory: "app-guides"
 title: "Chat Webhooks"
 tagline: "Find out how to configure chat webhooks in the LiveChat Developer Console."
-desc: "Use chat webhooks to get notified about the events happening during chats, for example, when there is an incoming chat, when someone sends a message, or when an agent tags a chat."
+desc: "Use chat webhooks to get notified about the events, for example, when there is an incoming chat or when an agent tags a discussion."
 ---
 
 ## Introduction
 
-This document explains how to configure the **Chat Webhooks** building block in <Placeholder id="DEVELOPER_CONSOLE_URL" />. It was written for everyone aiming to configure webhooks in their integrations **via user interface** instead of calling the LiveChat [Configuration API](/management/configuration-api/#webhooks) directly. The document assumes you're already familiar with the general idea of webhooks.
-
-The document applies to the so called **new chat webhooks** (v3.3 or later). It's impossible to configure the old chat webhooks (v3.2 or below) via Developer Console – you can only delete the already configured ones or [migrate them](#webhook-migration) to the current stable version. If you're curious about the differences between the old and new chat webhooks, [read this article](https://developers.livechat.com/updates/setting-up-webhooks-with-livechat-api).
+This document explains how to configure the **Chat Webhooks** building block in <Placeholder id="DEVELOPER_CONSOLE_URL" />. It was written for everyone aiming to configure webhooks in their integrations **via user interface** instead of calling the LiveChat [Configuration API](/management/configuration-api/#webhooks) directly. The document assumes you're already familiar with the general idea of webhooks. It applies to the **Chat webhooks** (v3.3 or later).
```

Comment:
I believe the info about the old versions is unnecessary now that v3.2 will not be available. That is why I also removed the section about Webhook migration, which is located below.